CHANGELOG.md

```
@@ -34,11 +43,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - Multiple strategies: `greedy`, `spread`
+  - Output modes: `list`, `bundle`, `json`
+- **Redaction Utilities**: New `tokmd-redact` crate centralizes BLAKE3-based path hashing
+- **CI Hyper-Testing**: Added proptest smoke tests, mutation testing, and fuzz testing workflows
+- **Integration Tests**: Comprehensive `analyze` command smoke tests
+- **Check-Ignore Command**: New `check-ignore` command explains why files are being ignored
+  - Delegates to `git check-ignore -v` for git-related ignores
+  - Shows `.tokeignore` and `--exclude` pattern matches
+  - Exit codes: 0=ignored, 1=not ignored
+- **Shell Completions**: New `completions` command generates shell completions for bash, zsh, fish, powershell, and elvish
 
 ### Changed
```

markdownlint-cli2 (0.20.0)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)